index.js

```diff
@@ -1,88 +1,239 @@
-module.exports = {
-  extends: '@antfu',
-  rules: {
-    'arrow-parens': ['error', 'always'],
-    'curly': ['error', 'all'],
-    'max-statements-per-line': 'off',
-    'no-cond-assign': 'off',
+import antfu from '@antfu/eslint-config'
+import { Alphabet } from 'eslint-plugin-perfectionist/alphabet'
 
-    'import/consistent-type-specifier-style': ['error', 'prefer-inline'],
-    'import/no-duplicates': ['error', {
-      'prefer-inline': true
-    }],
-    'import/order': ['error', {
-      groups: [
-        'builtin',
-        'external',
-        'internal',
-        'parent',
-        'sibling',
-        'index'
-      ],
-      pathGroups: [
-        {
-          pattern: '#*',
-          group: 'external'
-        },
-        {
-          pattern: '~*/**',
-          group: 'external',
-          position: 'before'
-        },
-        {
-          pattern: '@/composables/**',
-          group: 'parent',
-          position: 'before'
-        },
-        {
-          pattern: '@/components/**',
-          group: 'parent',
-          position: 'before'
-        },
-        {
-          pattern: '@/**',
-          group: 'internal',
-          position: 'after'
+/**
+ * @typedef {Parameters<typeof antfu>[1]} UserConfig
+ * @param {...UserConfig} userConfigs - User-defined ESLint configurations to extend or override
+ * @returns {ReturnType<typeof antfu>} A chainable ESLint config
+ */
+export default function globalbrain(...userConfigs) {
+  return (
+    antfu(
+      {
+        // `type: 'lib'` only adds [one rule](https://github.com/antfu/eslint-config/blob/0cd12cc90d2100798a5d8f5d51b34753b7be7f70/src/configs/typescript.ts#L165-L173),
+        // as our projects are mostly apps, we can just use the default app config,
+        // and either provide a separate config for lib projects,
+        // or manually enable the rule in the projects if needed
+
+        // TS/Vue features are auto-detected,
+        // but to avoid errors being thrown from our overrides,
+        // we need to explicitly enable them here.
+        typescript: true,
+        vue: true,
+
+        // We are more conservative on curly braces
+        // <https://github.com/antfu/eslint-config/blob/0cd12cc90d2100798a5d8f5d51b34753b7be7f70/src/configs/stylistic.ts#L54-L63>
+        lessOpinionated: true
+      },
+      {
+        rules: {
+          'max-statements-per-line': 'off'
```

Comment:
is this needed? style/max-statements-per-line is already disabled below and I don't think this is enabled by default